config.py

```
@@ -45,3 +45,8 @@
 MISTRAL_API_KEY   = "BJRjnwMi6jvna8zrfIvxyyr9Tf54HYgW"
 MISTRAL_MODEL     = "mistral-ocr-latest"
 
+# --- Datalab ------------------------------------------------------------------
+# Datalab OCR API settings
+DATALAB_API_KEY   = "vfaQi0_wwj5emvhyU2JtFJNtsLKSbkMsvaaD3nBwWLY"
```

Flake8 (7.2.0)

[error] multiple spaces before operator

(E221)